index.html

```diff
@@ -52,9 +52,19 @@ <h2>Where?</h2>
             We have a private channel on
             <a href="https://todogroup.org/community/comms/"
               >TODO Group's Slack</a
-            >, and a Zoom call every other Tuesday at 10:00 AM PT.
+            >, and a Zoom call every other Tuesday at 10:00 AM PT. Requirements
+            to participate:
           </p>
-
+          <ol>
+            <li>
+              Participants actively work within their company to adopt and
+              refine models that help unlock funding for Open Source projects.
+            </li>
+            <li>
+              Participants do this work agnostic of their company's own product
```

Comment:
Proposed replacement text:

"Participants are not involved in the development of funding platforms or funding-related products."

Reply:
@hartwork does this text read more clearly?

Reply:
@DuaneOBrien with me definitely.

Reply:
Updated in 8352d0f20ea4770ca26d74a5824d36ee6ed15289...cb28cdfcde748ea30f42c4aca0dfd3a6f7689dc2. 👍